lxd/resources/cpu.go

```diff
@@ -92,6 +92,16 @@ func ParseCpuset(cpu string) ([]int64, error) {
 	return cpus, nil
 }
 
+// ParseNumaNodeSet parses a `limits.cpu.nodes` into a list of NUMA node ids.
```

Comment:
Feels like this should be within `lxd/devices.go` alongside the scheduler function rather than in the `lxd/resources` package.

`lxd/resources` is used by external consumers like MAAS, so we don't want to have it contain stuff that's too LXD specific.